### ISSUE TITLE
ion-bottom-drawer Events

### ISSUE THREAD

**Issue by localhost9090** [toniantunovi/ion-bottom-drawer#32]
Does this component has events, such as slide up event slide down event or anything similar?

**Comment by Peerpal** [comment]
yes this is a big concern as in my implementation i need to know when the drawer is open or closed

**Comment by Peerpal** [comment]
the ion bottom drawer exposes the state changed event

`<ion-bottom-drawer
  [...]
  (stateChange)="changedState($event)"
>
[...]
</ion-bottom-drawer>`

so listen for the event then trigger your custom methods
